index.html

```diff
@@ -184,7 +182,9 @@ <h5>Sasanka Kundu</h5>
   <section id="services">
     <h1>Our Services</h1>
     <div data-aos="zoom-in" data-aos-duration="800" class="container">
-      <div class="responsive audio">
+
+      <!--Audio Therapy Card -->
+      <div class="responsive audio" id="card">

```

Comment:
The new code is not added on the first card so the animation dont work right.
line 187 should have: `<div data-aos="zoom-in" data-aos-duration="800" class="responsive audio aos-init aos-animate" id="card">`